packages/next/build/webpack/plugins/next-drop-client-page-plugin.ts

```diff
@@ -22,8 +22,7 @@ export class DropClientPage implements webpack.Plugin {
           const queue = new Set([mod])
           for (const module of queue) {
             if (isWebpack5) {
-              // @ts-ignore TODO: webpack 5 types
-              const incomingConnections = compilation.moduleGraph.getIncomingConnections(
+              const incomingConnections: Iterable<webpack.ModuleGraphConnection> = compilation.moduleGraph.getIncomingConnections(
```

Comment:
This should be auto-inferred based on the types in webpack without the need to assign it a specific type on the variable 🤔

Reply:
The problem here is that we need both webpack 4/5 types so this can't be changed until Next.js only supports webpack 5 in the future.